doc-site/docs/releasenotes/index.md

```diff
@@ -2,8 +2,29 @@
 title: Release Notes
 ---
 
+# Release Notes
+
 [Full release notes](https://github.com/hyperledger/firefly/releases)
 
+## [v1.3.3 - Mar 24, 2025](https://github.com/hyperledger/firefly/releases/tag/v1.3.3)
+
+What's New:
+- Add new method to provide reliable receipt storage for blockchain plugins
+- Changes in metrics:   
+    - Added new metrics for Data Exchange 
```

Comment:
What metrics are added and why? Should this be first in the list?

Reply:
Let me add them